### PR TITLE
pianoteq.standard-8: init at 8.1.1

### PR DESCRIPTION
**Pull request by ners**
###### Description of changes

In this PR I introduce a new package in the Pianoteq family: Standard 8.

I also cleaned up the code a bit and introduced desktop files for the software.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Why is ofborg not reporting any changed packages?

**Comment by ners** [comment]
@SuperSandro2000 that's a good question. These packages are also not shown on search.nixos.org

**Comment by ners** [comment]
@SuperSandro2000 maybe it's because it relies on a paywalled source download.

File: pkgs/applications/audio/pianoteq/default.nix [hunk 1]
```diff
@@ -41,18 +82,26 @@ let
             }
           fi
         done
+        install -Dm644 ${./pianoteq.svg} $out/share/icons/hicolor/scalable/apps/pianoteq.svg
```
[the remark this answers is not in the document]
Comment (ners) [reply pkgs/applications/audio/pianoteq/default.nix:85]: It's on the Modartt forum behind a URL that doesn't look too stable.

File: pkgs/applications/audio/pianoteq/default.nix [hunk 2]
```diff
@@ -41,18 +82,26 @@ let
...
+        for size in 16 32 64 128 256 512 1024; do
+          dir=$out/share/icons/hicolor/"$size"x"$size"/apps
+          mkdir -p $dir
+          inkscape -w $size -h $size ${./pianoteq.svg} -o $dir/pianoteq.png
```
[the remark this answers is not in the document]
Comment (ners) [reply pkgs/applications/audio/pianoteq/default.nix]: I've replaced it with librsvg.

**Comment by ners** [comment]
@SuperSandro2000 could you have another look?

**Pull request merged**